fixtures/chauvet-dj/megastrobe-fx12.json

```diff
@@ -0,0 +1,309 @@
+          "type": "ShutterStrobe",
+          "shutterEffect": "Closed"
+        }
+      ]
+    },
+    "Shutter Speed": {
+      "capability": {
+        "type": "Generic",
+        "comment": "Sensitivity OR Shutter Speed"
+      }
+    },
+    "Pixels": {
+      "capability": {
+        "type": "Generic",
+        "comment": "Pixel, Program Speed, or Sound Active Auto"
+      }
+    },
+    "Auto Programs": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 48],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [49, 64],
+          "type": "Effect",
+          "effectName": "Auto Program 1"
+        },
+        {
+          "dmxRange": [65, 80],
+          "type": "Effect",
+          "effectName": "Auto Program 2 "
+        },
+        {
+          "dmxRange": [81, 96],
+          "type": "Effect",
+          "effectName": "Auto Program 3"
+        },
+        {
+          "dmxRange": [97, 112],
+          "type": "Effect",
+          "effectName": "Auto Program 4"
+        },
+        {
+          "dmxRange": [113, 128],
+          "type": "Effect",
+          "effectName": "Auto Program 5 "
+        },
+        {
+          "dmxRange": [129, 144],
+          "type": "Effect",
+          "effectName": "Auto Program 6"
+        },
+        {
+          "dmxRange": [145, 160],
+          "type": "Effect",
+          "effectName": "Auto Program 7"
+        },
+        {
+          "dmxRange": [161, 176],
+          "type": "Effect",
+          "effectName": "Auto Program 8"
+        },
+        {
+          "dmxRange": [177, 192],
+          "type": "Effect",
+          "effectName": "Auto Program 9"
+        },
+        {
+          "dmxRange": [193, 208],
+          "type": "Effect",
+          "effectName": "Auto Program 10"
+        },
+        {
+          "dmxRange": [209, 224],
+          "type": "Effect",
+          "effectName": "Auto Program 11"
+        },
+        {
+          "dmxRange": [225, 240],
+          "type": "Effect",
+          "effectName": "Auto Program 12"
+        },
+        {
+          "dmxRange": [241, 255],
+          "type": "Effect",
+          "effectName": "Sound Active",
+          "soundControlled": true
+        }
+      ]
+    },
+    "Dimmer 1": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Dimmer 2": {
+      "name": "Dimmer",
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Dimmer 3": {
+      "name": "Dimmer",
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Dimmer 4": {
+      "name": "Dimmer",
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Dimmer 5": {
+      "name": "Dimmer",
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Dimmer 6": {
+      "name": "Dimmer",
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Shutter Effects": {
+      "capability": {
+        "type": "Generic"
+      }
+    },
```

Comment:
I'm really not sure how to interpret the manual for this channel.

Reply:
Hmm, since the manual does not mention any capabilities, we can't be more precise here. But please add a `helpWanted` message here.